docs/detections/detections-ui-exceptions.asciidoc

```diff
@@ -253,3 +256,23 @@ Creates an exception that excludes all LFC-signed trusted processes:
 
 [role="screenshot"]
 image::images/nested-exp.png[]
+
+[float]
+[[manage-exceptions]]
+=== View and manage exceptions
+
+The Exception lists table enables you to view and manage all exceptions that have been assigned to rules. To view the Exception lists table, go to *Detections* -> *Manage detection rules*, then select the *Exception Lists* tab.
+
+[role="screenshot"]
+image::images/exception-list.png[]
+
+The table displays each Exception list on an individual row, with the most recently created list at the top. Each list contains information such as the number of rules the exception is assigned to, the name of the rule(s) assigned to the exception, the date created, the date last edited, and options to export or delete the list. Use the Search bar to enter a KQL query to filter the lists by `name`, `created_by`, or `list_id`.
```

Comment:
Super nit: should we be consistent with lingo of assigning lists to things vs assigning things to lists? The former is more accurate in terms of describing the relationship code wise.

```suggestion
The table displays each Exception list on an individual row, with the most recently created list at the top. Each list contains information such as the number of rules the exception is assigned to, the name of the rule(s) the exception list is assigned to, the date created, the date last edited, and options to export or delete the list. Use the Search bar to enter a KQL query to filter the lists by `name`, `created_by`, or `list_id`.
```